modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -87,13 +91,24 @@ spec:
    logLevel: debug <3>
    overlaySize: 8G <4>
    logSizeMax: "-1" <5>
+   defaultRuntime = "crun" <6>
 ----
 <1> Specifies the machine config pool label.
 <2> Optional: Specifies the maximum number of processes allowed in a container.
 <3> Optional: Specifies the level of verbosity for log messages.
 <4> Optional: Specifies the maximum size of a container image.
 <5> Optional: Specifies the maximum size allowed for the container log file. If
 	set to a positive number, it must be at least 8192.
+<6> Optional: Specifies the container runtime to deploy to new containers. The default is `runc`.
+
+.Prerequisite
+
+* To use the `defaultRuntime` parameter, you must enable the  `TechPreviewNoUpgrade` feature set by using the feature gates.
```

Comment:
this is not technically correct, though functionally it doesn't matter. `defaultRuntime` parameter is available on all clusters. on clusters without the TechPreviewNoUpgrade flag, the only option is empty string "' or `runc`. On clusters with it, `crun` is an additional option available